bounties_api/std_bounties/management/commands/bounties_subscriber.py

```diff
@@ -18,8 +18,135 @@
 class Command(BaseCommand):
     help = 'Listen to SQS queue for contract events'
 
+    def handle_message(self, message):
+        receipt_handle = message['ReceiptHandle']
+        message_attributes = message['MessageAttributes']
+
+        event = message_attributes['Event']['StringValue']
+        bounty_id = int(message_attributes['BountyId']['StringValue'])
+        fulfillment_id = int(
+            message_attributes['FulfillmentId']['StringValue'])
+        message_deduplication_id = message_attributes['MessageDeduplicationId']['StringValue']
+        transaction_from = message_attributes['TransactionFrom']['StringValue']
+        transaction_hash = message_attributes['TransactionHash']['StringValue']
+        event_timestamp = message_attributes['TimeStamp']['StringValue']
+        contract_method_inputs = json.loads(
+            message_attributes['ContractMethodInputs']['StringValue'])
+        event_date = datetime.datetime.fromtimestamp(int(event_timestamp))
+
+        # If someone uploads a data hash that is faulty, then we want to blacklist all events around that
+        # bounty id. We manage this manually
+        if redis_client.get('blacklist:' + str(bounty_id)):
+            redis_client.set(message_deduplication_id, True)
+            sqs_client.delete_message(
+                QueueUrl=settings.QUEUE_URL,
+                ReceiptHandle=receipt_handle,
+            )
+            return False
+
+        event_defaults = {
+            'bounty_id': bounty_id,
+            'fulfillment_id': fulfillment_id if fulfillment_id != -1 else None,
+            'transaction_from': transaction_from,
+            'contract_inputs': contract_method_inputs,
+            'event_date': event_date,
+        }
+
+        logger.info('get_or_create with defaults: {}'.format(event_defaults))
```

Comment:
I'm logging lots of extra info here temporarily, and moved the event creation up higher in the function